apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -45,6 +45,8 @@ class AnimationPicker extends React.Component {
 
     // Provided via Redux
     visible: PropTypes.bool.isRequired,
+    isBackground: PropTypes.bool,
+    isSpriteLab: PropTypes.bool,
```

Comment:
similar to what we did with `<AnimationPickerBody/>`, these props should be calculated by the parent component (`<P5LabView/>`). then we can just pass the new props through without this component needing to understand how to calculate these props, and the parent already has similar logic (e.g., if i'm in spritelab, do this, if i'm in gamelab, do something else)